scripts/communityModules/chat/FloofChat.html

```diff
@@ -70,6 +70,40 @@
 
     //Start George Function
     //Function provided by George Deac
+  
+  // Note soundcloud and vimeo integration is disabled since we cannot support mp4/mp3 encoding at this time
+	// Soundcloud oembed jQuery api script modified for embdeding by class name
+	/*
+	document.addEventListener("DOMContentLoaded", function(event) {
+		(function ($) {
+			$.fn.scembed = function(){
+			var datasource  = 'https://soundcloud.com/oembed';
+			return this.each(function () {
+				var container = $(this);
+				var mediasource = $(container).attr("sc_url");
+				var params = 'url=' + mediasource + '&format=json&iframe=true&maxwidth=480&maxheight=120&auto_play=false&show_comments=false';
+				$.ajaxopts = $.extend($.ajaxopts, {
+									url: datasource,
+									data: params,
+									dataType: 'json',
+									success: function (data, status, raw) {
+										$(container).html(data.html);
+									},
+									error: function (data, e1, e2) {
+										$(container).html("Can't retrieve player for " + mediasource);
+									},
+								});
+				$.ajax($.ajaxopts);
+				});
+			};
+		})(jQuery);
+
+		$(function(){
+			$("div.sc-embed").scembed();
+		});
+	});
+*/
```

Comment:
Indentation

Reply:
Does this disabled code need to be here? Could it not be added when support is added?

Reply:
Can be removed